boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -101,6 +109,8 @@ static mut BME280: Option<
     >,
 > = None;
 static mut CCS811: Option<&'static capsules_extra::ccs811::Ccs811<'static>> = None;
+#[cfg(feature = "atecc508a")]
+static mut ATECC508A: Option<&'static capsules_extra::atecc508a::Atecc508a<'static>> = None;
```

Comment:
The fact that this is a global static and the associated `unsafe`s to access it are not great. Why does `static_init` not work for this board?

Reply:
We use the global statics to access the data from the test infrastructure. It's the same thing we do with OpenTitan and all boards that support unit tests